Makefile

```diff
@@ -1,15 +1,28 @@
-all: install compile-sass quality test
+all: clean install requirements quality test test-python
+
+clean: ## delete generated byte code and coverage reports
+	find . -name '*.pyc' -delete
+	find . -name '__pycache__' -type d -exec rm -rf {} ';' || true
+	rm -rf .coverage
+	rm -rf assets
+	rm -rf pii_report
 
 install-test:
 	pip install -q -r requirements/test.txt
 
 install: install-test
 
+requirements: ## install development environment requirements
+	pip install -q -r requirements/dev.txt
+
 quality:
 	./scripts/quality.sh
 
-test:
-	./scripts/test.sh
+test-python: clean
+	python -m coverage run --rcfile=.coveragerc test.py $1 --noinput
```

Comment:
Can we run this with pytest instead as we have been in other repos?

Reply:
In repos with django where tests are written in `test.py`, we can't use `pycharm` because pycharm can't detect those test, it looks for `*_test.py` or `test_*.py` pattern.

Reply:
Why don't we rename the file to test_edx_zoom.py?